generator/explores/ping_explore.py

```diff
@@ -22,6 +41,7 @@ def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
                 "always_filter": {
                     "filters": self.get_required_filters("base_view"),
                 },
+                "queries": deepcopy(PingExplore.queries),
```

Comment:
Why the deep copy?

Reply:
Good question, maybe @fbertsch can answer?
I've simply copied it from other existing example queries: https://github.com/mozilla/lookml-generator/blob/8db3ebc21e4daaeb861f75e943cc98e6c8d3ce16/generator/explores/events_explore.py#L52

Reply:
I talked to Frank about this. Apparently, the `lkml` library that we are using mutates the data.